tests/core/test_check_manifest.py

```diff
@@ -95,8 +94,7 @@ def test_check_allowed_filepaths_fail(scroot, monkeypatch):
     os.chdir(cwd)
 
     env = {
-        'SC_VALID_PATHS': os.path.join(scroot, 'third_party', 'pdks'),
-        'SCPATH': os.environ['SCPATH']
+        'SC_VALID_PATHS': os.path.join(scroot, 'third_party', 'pdks')
```

Comment:
is this used somewhere?

Reply:
Seems it's fine to remove `SC_VALID_PATHS` too.